### PR TITLE
Optimizely page view tracking

### LINKED ISSUE #9788: Track page views in Optimizley for Sty/Article pages
**Is your feature request related to a problem? Please describe.**
Track page views for each variation in optimzley
The page already fires of an ATI event when the page loads  

**Describe the solution you'd like**
Create a new event in Optimizley to capture page views
Fire the event when the page loads. 

We can use the Optimizely object which has a track function, we can access this object with UseContext, issue explaining [this](https://github.com/optimizely/react-sdk/issues/78).

Here is an example for where the track function is used in our click tracker https://github.com/bbc/simorgh/blob/71ecd0c7f15eb1dafa496aba463b3fdf029b4b2f/src/app/hooks/useClickTrackerHandler/index.jsx#L61-L63

Try and follow what we've done for ATI analytics [here](https://github.com/bbc/simorgh/blob/latest/src/app/containers/ATIAnalytics/index.jsx), where we use this component on each page we want to track a page view, see the [Article page](https://github.com/bbc/simorgh/blob/71ecd0c7f15eb1dafa496aba463b3fdf029b4b2f/src/app/pages/ArticlePage/ArticlePage.jsx#L170) as an example.

We don't need any separation for STY and Article page reports, so page views can be combined.

**Describe alternatives you've considered**

**Testing notes**
View a page running an experiment
View each variation on the page 
Check the page views in Optimizley matches the expected value for each variation
We can disable it on the test environment, and a specific service can be chosen to test the page views are working correctly locally.
Look at disabling a specific E2E which may affect the page view event being sent.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

### PR DESCRIPTION
**Pull request by amoore108**
Resolves #9788

**Overall change:**
Adds page view tracking to Story and Article page types, using the `page_views` Optimizely identifier. This tracking event will only occur under the following criteria:
- Optimizely is available
- It is a canonical Story / Article page 

**Code changes:**

- Adds a new component called `<OptimizelyPageViewTracking />`, which is used in a similar fashion to the existing ATI/Chartbeat components in the Story and Article pages

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/OptimizelyPageViewTracking/index.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { useState, useContext, useEffect } from 'react';
+import { OptimizelyContext } from '@optimizely/react-sdk';
+import { RequestContext } from '#contexts/RequestContext';
+import useMediaQuery from '#app/hooks/useMediaQuery';
+import { GEL_GROUP_3_SCREEN_WIDTH_MAX } from '@bbc/gel-foundations/dist/breakpoints';
+
+const OptimizelyPageViewTracking = () => {
+  const { isAmp } = useContext(RequestContext);
+  const { optimizely } = useContext(OptimizelyContext);
+  const [mobile, setMobile] = useState(false);
```
Comment (HarveyPeachey) [inline src/app/containers/OptimizelyPageViewTracking/index.jsx]: Just to carry over from slack, I'm happy for this to be removed

**Review by HarveyPeachey — APPROVED** [review]
LGTM -

**Review by Louis-Matsika — APPROVED** [review]

**Review by andrewscfc — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Testing:

Checked using the following URL's - 
* http://localhost:7080/bengali/articles/c6p3yp5zzmeo
* http://localhost:7080/igbo/afirika-23252735
* http://localhost:7080/igbo/articles/cr1lw620ygjo

Screenshot 2022-01-27 at 14 21 58

I just need to check Optimizely to see if the Pageviews have come through but currently I am having an issue with experiments not loading.. It should be a quick check once its back and the ticket is good to go unless there are any issues..

Screenshot 2022-01-27 at 15 02 37

**Comment by paruchurisilpa** [comment]
@amoore108 I am looking at the pageviews for Igboservice, I have viewed 2 URL:'s mentioned above but I only see 1 page view.

Screenshot 2022-01-28 at 08 32 04

**Comment by HarveyPeachey** [comment]
> @amoore108 I am looking at the pageviews for Igboservice, I have viewed 2 URL:'s mentioned above but I only see 1 page view.
> 

You might need to change it to total conversions instead of unique conversions as it will only show 1 for a session when set to unique conversions I think @paruchurisilpa

**Pull request merged**